src/main/java/Noisy/Ui.java

```diff
@@ -0,0 +1,59 @@
+package Noisy;
+
+public class Ui {
+
+    public void printWelcome() {
+        String welcomeMessage = "____________________________________________________________\n"
+                + " Hello! I'm Noisy.Noisy\n"
+                + " What can I do for you?\n"
+                + "____________________________________________________________\n"
+                + " Bye. Hope to see you again soon!\n"
+                + "____________________________________________________________";
```

Comment:
I like how you break the string into a few lines, which increase the readability.